Objects/ComingObjects/ComingObject.gd

```diff
@@ -5,14 +5,16 @@ signal timeout
 func _ready():
 	connect("timeout",self,"_on_timeout")
```

Comment:
Ok, you connect _on_timeout method but _on_timeout method doing nothing... Why?

Reply:
This is not my code. It was there before i think. I didn't need to add any timer

Reply:
This is not my code. It was there before i think. I didn't need to add any timer

Reply:
actually but we can remove _ready() method, timeout signal and _on_timeout method

Reply:
actually but we can remove _ready() method, timeout signal and _on_timeout method

Reply:
actually but we can remove _ready() method, timeout signal and _on_timeout method

Reply:
We can remove _on_timeout, but we cant remove signal. it is responsible for entering the tavern. When we delete signal, tavern just passes away and disappears.